src/axom/inlet/SphinxWriter.cpp

```diff
@@ -114,6 +124,16 @@ void SphinxWriter::documentContainer(const Container& container)
     currContainer.description = sidreGroup->getView("description")->getString();
   }
 
+  // Bail out if this is a collection of primitives -
+  // it doesn't make sense to document each individual primitive
+  // FIXME: Implement something analogous to the logic for struct collections that displays the
+  // "schema" for the primitive elements of the collection
+  if(isCollectionGroup(container.name()) &&
+     !sidreGroup->hasView(detail::STRUCT_COLLECTION_FLAG))
+  {
+    return;

```

Comment:
Does the user end up knowing this is a collection of primitives from the outputted documentation?

Reply:
It's not extremely clear (see https://axom.readthedocs.io/en/feature-essman-nested_docs/axom/inlet/docs/sphinx/mfem_coefficient_nested_documentation.html#attrs) but I can clean this up in a follow-up once we decide what that should look like (see also https://github.com/LLNL/axom/pull/482#pullrequestreview-598879079)